youboraplugin/src/main/java/com/kaltura/playkit/plugins/youbora/pluginconfig/YouboraConfig.java

```diff
@@ -58,13 +107,42 @@ public class YouboraConfig {
 
     private boolean isAutoStart = true;
 
-    private boolean isAutoDetectBackground = true;
+    @SerializedName(value="isAutoDetectBackground", alternate={"autoDetectBackground"})
+    private boolean isAutoDetectBackground = true; // backward compatible
+
+    @SerializedName(value="isEnabled", alternate={"enabled"})
+    private boolean isEnabled = true; // backward compatible
+
+    @SerializedName(value="isForceInit", alternate={"forceInit"})
+    private boolean isForceInit; // backward compatible
+
+    @SerializedName(value="isOffline", alternate={"offline"})
+    private boolean isOffline; // backward compatible
 
-    private boolean isEnabled = true;
+    /**
+     * Enabling this option enables the posibility of getting the /start request later on the view,
+     * making the flow go as follows: /init is sent when the player starts to load content,
+     * then when the playback starts /joinTime event will be sent, but with the difference of no
+     * /start request, instead it will be delayed until all the option keys from
+     * {@link #pendingMetadata} are not <b>null</b>, this is very important, since an empty string
+     * is considered a not null and therefore is a valid value.
+     */
+    private boolean waitForMetadata;
 
-    private boolean isForceInit;
+    private ArrayList<String> pendingMetadata;
 
-    private boolean isOffline;
+    @SerializedName(value="sessionMetrics", alternate={"session"})
+    private HashMap<String, String> sessionMetrics;
```

Comment:
Fix this with Session Class and put the metrics inside 

"session": [
        "metrics": [
            "metricskey": ["test": 66],
            "metricskey2": "test",
            "metricskey3": 2
            // ...
        ]
    ]

Reply:
fixed